### ISSUE TITLE
Hide Running Jobs on Activity Page

### ISSUE THREAD

**Issue by dirkniblick** [rundeck/rundeck#2816]
Issue type: Enhancement Request

**My Rundeck detail**

* Rundeck version: 2.9.3-1
* install type: RPM
* OS Name/version: CentOS 7
* DB Type/version: MySQL

**Expected Behavior**
The activity page should be a stress-free place to view job executions.

**Actual Behavior**
When I'm trying to locate a failed job or whatnot, the table on the Activity page is constantly jumping around.  With jobs running every few seconds, the table repeatedly resizing to make room for the bars and then snapping back is, to be frank, obnoxious.  Providing the ability to disable running jobs from appearing on the activity page or just have them not if you're beyond the first page would be less stress inducing when you're trying to find a failed job (and maybe the jobs around it).

**Comment by ahonor** [comment]
+1 

There should be a boolean that can turn off the running jobs

**Comment by dirkniblick** [comment]
@ahonor, you could add a checkbox on the screen or something, but honestly it seems like it should be part of the "Auto refresh" capability.  It'd also be nice if the first item in the list didn't change when new executions were added (i.e., the page/query remembered the starting execution).  It's frustrating when you're changing pages and the executions hop around because the first item way in the front keeps sliding down.  (e.g.,"1, 2, 3..." *move a page forward* "4, 5 - wait, I already counted these.")

**Comment by ahonor** [comment]
@dirkniblick I agree with your sentiments. I just replicated that behavior and it is pretty annoying. So you think it should not be a new toggle?

**Comment by dirkniblick** [comment]
Others might disagree with me, but having running jobs appear at the top of the "stack" and have them pushed down into the list seems like an all or nothing thing to me.  I can't imagine why I'd want one but not the other.

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
